parent/pom.xml

```diff
@@ -65,6 +65,8 @@
 		<jbosstools.test.jre.7></jbosstools.test.jre.7>
 		<jbosstools.test.jre.8></jbosstools.test.jre.8>
 
+
+
```

Comment:
I disagree wholeheartedly with this change. Whitespace is too eurocentric and racially biased.